gradle.properties

```diff
@@ -32,3 +32,4 @@ apachePassWord=
 signEnabled=false
 
 org.gradle.warning.mode=none
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
Why you add this param ?

Reply:
> Why you add this param ?

@xwm1992  oh, I will delete that. just for debugging.